util/CommonConstants.go

```diff
@@ -27,6 +27,8 @@ const RETRYCOUNT = 10
 const WORKINGDIR = "/devtroncd"
 const LOCAL_BUILDX_LOCATION = "/var/lib/devtron/buildx"
 const LOCAL_BUILDX_CACHE_LOCATION = LOCAL_BUILDX_LOCATION + "/cache"
+const BUILD_CACHE_FOLDER_PVC = "/devtroncd-cache"
```

Comment:
do we need these constants now ??

Reply:
sorry forgot to remove them, removed now since we don't need them